conda/recipes/libcudf/meta.yaml

```
@@ -19,22 +19,29 @@ build:
     - CMAKE_C_COMPILER_LAUNCHER
     - CMAKE_CXX_COMPILER_LAUNCHER
     - CMAKE_CUDA_COMPILER_LAUNCHER
-    - SCCACHE_S3_KEY_PREFIX=libcudf-aarch64 # [aarch64]
-    - SCCACHE_S3_KEY_PREFIX=libcudf-linux64 # [linux64]
-    - SCCACHE_BUCKET=rapids-sccache
-    - SCCACHE_REGION=us-west-2
-    - SCCACHE_IDLE_TIMEOUT=32768
+    # TODO: remove "gha" suffix before PR is merged
+    - SCCACHE_S3_KEY_PREFIX=libcudf-aarch64-gha # [aarch64]
+    - SCCACHE_S3_KEY_PREFIX=libcudf-linux64-gha # [linux64]
```

Leave this suggestion open until we're ready to merge, then apply it.
```suggestion
    - SCCACHE_S3_KEY_PREFIX=libcudf-aarch64 # [aarch64]
    - SCCACHE_S3_KEY_PREFIX=libcudf-linux64 # [linux64]
```

---

resolved in 85b1ee1